docs/content/preview/explore/ysql-language-features/foreign-data-wrappers.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Foreign Data Wrappers
+linkTitle: Foreign Data Wrappers
+description: Foreign Data Wrappers in YSQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: explore-ysql-language-features-foreign-data-wrappers
+    parent: explore-ysql-language-features
+    weight: 100
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how to use foreign data wrappers.
+
+## Overview
+
+A foreign data wrapper is a library that you can use to access and interact with an external data (foreign data) source. They allow you to query foreign objects from remote servers as if they were local objects.
+
+In order to access foreign data, we first need to create the foreign data wrapper. Then, we need to create a foreign server, which specifies how to connect to the external data source. We may also need to create a user mapping which defines the mapping of a specific user to authorization credentials in the foreign server. Lastly, we need to create foreign tables, which represent the structure of the data on the external source. 
+
+### CREATING FOREIGN DATA WRAPPERS
```

Comment:
This should be sentence case, since it's not actually the statement name. Also, use imperative headings ("create a...") rather than gerund headings ("creating a...") wherever possible.
```suggestion
## Create a foreign data wrapper
```